.eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
   rules: {
     'no-var': 'error',
     'no-console': 'off',
+    'no-unused-vars': 'off', // this is turned off because of the lack of scope analysis on imported types (https://github.com/typescript-eslint/typescript-eslint/issues/1856)
```

Comment:
I think if you use their recommended config you don't have to explicitly disable this (they have their own variant of `no-unused-vars` that *is* type aware).

From their [readme](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin):

```
{
  "extends": ["plugin:@typescript-eslint/recommended"]
}
```

Reply:
I did this and fixed all the errors, I need to go add typing information to the imports, will do that hopefully tonight or tomorrow.